### ISSUE TITLE
0.18.0 Release Plan

### ISSUE THREAD

**Issue by jorisroovers** [jorisroovers/gitlint#248]
This issue will be used to track any overarching discussion and work related to `0.18.0`.

# Scope

**As always, everything discussed in this issue is subject to sudden change at my sole discretion. No promises or hard commitments :-)**

All related issues will be tagged with the [0.18.0 Milestone](https://github.com/jorisroovers/gitlint/milestone/5).

We also maintain a [Kanban board on Github Projects](https://github.com/users/jorisroovers/projects/1/views/1), items marked as TODO are things I'd like to work on for the next release (but again, subject to change).

# Timeline

As always, no specific timeline, but probably in the new year (2022).

**Comment by jorisroovers** [comment]
Going to give the new-ish github projects a try as an experiment and attempt to do some better project planning.
In particular, would like to move and clean up the open items on https://github.com/jorisroovers/gitlint/wiki/Roadmap as well as the list under **Scope** above, as neither are very representative of what's currently being worked on.

https://github.com/users/jorisroovers/projects/1

Will work on this over next week or so.

**Comment by jorisroovers** [comment]
Just released 0.18.0 🎉 

Did hit an issue with the docker image build for 0.18.0, I'll follow up on that soon - I think something changed on the docker or git side.
```sh
DEBUG: gitlint.git ('log', '-1', '--pretty=%H')
An error occurred while executing 'git log -1 --pretty=%H': b"fatal: detected dubious ownership in repository at '/repo'\nTo add an exception for this directory, call:\n\n\tgit config --global --add safe.directory /repo"
```

@harens Can you publish for macOS? Super thanks 🙌

**Comment by harens** [comment]
> @harens Can you publish for macOS? Super thanks 🙌

All done! Thanks for another release @jorisroovers.

**Comment by jorisroovers** [comment]
Awesome, thanks @harens 🫶

**Comment by jorisroovers** [comment]
The docker issue was fixed in #365 - the 0.18.0 docker image is now available on dockerhub! https://hub.docker.com/r/jorisroovers/gitlint/tags

Closing this release out!
